examples/09_spatio_temporal/03_geographic_coordinates.py

```diff
@@ -0,0 +1,36 @@
+"""
+Working with spatio-temporal lat-lon fields
+-------------------------------------------
+
+In this example, we demonstrate how to generate a spatio-temporal
+random field on geographical coordinates.
+
+First we setup a model, with ``latlon=True`` and ``time=True``,
+to get the associated spatio-temporal Yadrenko model.
+
+In addition, we will use the earth radius provided by :any:`EARTH_RADIUS`
+as ``geo_scale`` to have a meaningful length scale in km.
+
+To generate the field, we simply pass ``(lat, lon, time)`` as the position tuple
+to the :any:`SRF` class.
+
+The anisotropy factor of `0.1` (days/km) will result in a time length-scale of `100` days.
```

Comment:
I think I would find it more intuitive, if a list of length scales is provided, instead of an anisotropy-factor.

Reply:
Nevermind, it works... but why?!

Reply:
Wow, just a solid logic for len_scale and anis implemented in the past. Will use `len_scale=[1000, 100]` in this example.

Reply:
Done